muselog/__init__.py

```diff
@@ -30,6 +30,10 @@ def setup_logging(root_log_level: Optional[str] = None,
 
     root_logger = logging.getLogger()
     root_logger.setLevel(root_log_level)
+    default_stdout_handler = None
+
+    if len(root_logger.handlers) > 0:
+        default_stdout_handler = root_logger.handlers[0]
```

Comment:
This seems very shaky. How do you know the first handler is the correct one?

Reply:
Would it make sense for muselog to iterate over all the log handlers from a given application and remove them as part of the setup process?

Reply:
By default, the `StreamHandler` is always set as the first handler by python logger.

Reply:
@andela-sjames If that's true why do we not see the same issue with non-tornado apps like rex?

Reply:
Python adds the default if it doesn't find any handler at first, my guess will be that from the setup, it finds the handler already configure and does nothing. 

On the other hand the framework can also override this behaviour to add theirs. Also, since we are using tornado for web, according their documentation page, `https://www.tornadoweb.org/en/stable/log.html` the framework uses streams as it's default logger. This would explain why it is sort of different for web.

This came to my attention when the test failed at first, the same reason I check if there is a default handler before removing it from the index, the idea is you can't remove the default unless you already have a second handler added to it.

Reply:
Can you add a pydoc explaining the selection algorithm? We should be clear how `add_console_handler` works. If need be, we can add an option to supply a default handler. Probably won't need it now.